### PR TITLE
Update API docs for eu-west

### PR DESCRIPTION
**Pull request by electron0zero**
added API URL for stacks that are in Europe Region

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -4,8 +4,26 @@ Synthetic Monitoring API
 This document describes the Synthetic Monitoring API. All the entry
 points return results formatted as JSON objects.
 
+## API URL
+
+Based on region of your Grafana Cloud stack, you need to use a diffrent API URL
```
Comment (rdubrock) [inline docs/API.md:9]:
```suggestion
Based on region of your Grafana Cloud stack, you need to use a different API URL
```

**Review by rdubrock — APPROVED** [review]

File: docs/API.md [hunk 2]
```diff
@@ -4,8 +4,26 @@ Synthetic Monitoring API
...
+**United States Region**
+
+Synthetic Monitoring API is hosted at [https://synthetic-monitoring-api.grafana.net](https://synthetic-monitoring-api.grafana.net)
+
+**Europe Region**
+
+Synthetic Monitoring API is hosted at [https://synthetic-monitoring-api-eu-west.grafana.net](https://synthetic-monitoring-api-eu-west.grafana.net)
+
+
+If you are unsure about region of your stack, check **Synthetic Monitoring -> Config** page.
```
Comment (rdubrock) [inline docs/API.md:20]:
```suggestion
If you are unsure about the region of your stack, check **Synthetic Monitoring -> Config** page. 
```

File: docs/API.md [hunk 3]
```diff
@@ -4,8 +4,26 @@ Synthetic Monitoring API
...
+
+**Europe Region**
+
+Synthetic Monitoring API is hosted at [https://synthetic-monitoring-api-eu-west.grafana.net](https://synthetic-monitoring-api-eu-west.grafana.net)
+
+
+If you are unsure about the region of your stack, check **Synthetic Monitoring -> Config** page. 
+API URL for your Grafana Cloud instance will be listed under **Backend address** 
+
+> NOTE: You need to be `Admin` in to access Synthetic Monitoring config page
```
Comment (electron0zero) [inline docs/API.md:23]:
```suggestion
> NOTE: You need to be `Admin` in Grafana to access Synthetic Monitoring config page
```

**Review by rdubrock — COMMENTED** [review]
Looks great!

**Pull request merged**